src/main/java/org/jabref/gui/entryeditor/EntryEditor.java

```diff
@@ -179,7 +181,29 @@ public void keyReleased(java.awt.event.KeyEvent e) {
             public void keyPressed(java.awt.event.KeyEvent e) {
                 //We need to consume this event here to prevent the propgation of keybinding events back to the JFrame
 
-                e.consume();
+                Optional<KeyCode> keyCode = Arrays.stream(KeyCode.values()).filter(k -> k.getName().equals(e.getKeyText(e.getKeyCode()))).findFirst();
+                keyCode.ifPresent(code -> {
+                    KeyEvent event = new KeyEvent(e.getSource(), null, KeyEvent.KEY_PRESSED, "", "", code, e.isShiftDown(), e.isControlDown(), e.isAltDown(), e.isMetaDown());
+
+                    Optional<KeyBinding> keyBinding = Globals.getKeyPrefs().mapToKeyBinding(event);
+                    if (keyBinding.isPresent()) {
+
+                        switch (keyBinding.get()) {
+                        case CUT:
+                        case COPY:
+                        case PASTE:
+                        case CLOSE_ENTRY_EDITOR:
+                        case DELETE_ENTRY:
+                        case SELECT_ALL:
+                            e.consume();
```

Comment:
Yes, unfortunately. Although the event is consumed it is still catched by the swing side and executed.
 I have found no way to stop this.